modules/ROOT/partials/plugin-files/plugin-file-list-tinycomments.adoc

```diff
@@ -1,43 +1,18 @@
-....
+.Base js file
+[source, js]
+----
+./plugins/tinycomments/plugin.js
+----
+
+.css
+[source, js]
+----
 ./plugins/tinycomments/css/tinycomments.css
 ./plugins/tinycomments/js/tinycomments-sidebar.js
```

Comment:
This is also an extra file part of the lazy loading process normally but can be pre-bundled. Should be it's own section.

Reply:
Separated this into its own section as `.other js files` as discussed.